src/test/java/space/obminyashka/items_exchange/rest/ImageControllerIntegrationTest.java

```diff
@@ -146,12 +146,14 @@ void getByAdvertisementId_shouldReturn404WhenAdvertisementIsNotExist() throws Ex
     @WithMockUser("admin")
     @Test
     void deleteImages_shouldThrow400WhenImageIdNotExist() throws Exception {
+        when(advertisementService.isUserHasAdvertisementWithId(any(UUID.class), eq("admin"))).thenReturn(true);
```

Comment:
that can be moved to mocksInit method when(advertisementService.isUserHasAdvertisementWithId(advertisementId, "admin")).thenReturn(true);